startup_scripts/000_users.py

```diff
@@ -21,17 +20,4 @@
       Token.objects.create(user=user, key=user_details['api_token'])
 
     yaml_permissions = user_details.get('permissions', [])
-    if yaml_permissions:
-      subject = user.user_permissions
-      subject.clear()
-      for yaml_permission in yaml_permissions:
-        if '*' in yaml_permission:
-          permission_filter = '^' + yaml_permission.replace('*','.*') + '$'
-          permissions = Permission.objects.filter(codename__iregex=permission_filter)
-          print("  ⚿ Granting", permissions.count(), "permissions matching '" + yaml_permission + "'")
-        else:
-          permissions = Permission.objects.filter(codename=yaml_permission)
-          print("  ⚿ Granting permission", yaml_permission)
-
-        for permission in permissions:
-          subject.add(permission)
+    set_perms_from_yaml(subject=user.user_permissions, permissions=yaml_permissions)
```

Comment:
Following the other suggestion, this would become that:

```suggestion
    set_permissions(user.user_permissions, yaml_permissions)
```

Reply:
Done